packages/studio/src/components/Highlighters.tsx

```diff
@@ -1,20 +1,54 @@
-import { CSSProperties, useEffect, useMemo } from "react";
-import useActiveComponentName from "../hooks/useActiveComponentName";
+import { CSSProperties, useMemo } from "react";
+import { useComponentNames } from "../hooks/useActiveComponentName";
 import useStudioStore from "../store/useStudioStore";
 import { ComponentStateKind } from "@yext/studio-plugin";
+import DOMRectProperties from "../store/models/DOMRectProperties";
 
 /**
- * Highlights the current activeComponentRect on the page.
+ * Generates Highlighters for all selected components.
```

Comment:
Just realized that this would cause overlapping name labels - working on fixing rn!